editor/components/inserter/index.js

```diff
@@ -36,8 +41,12 @@ class Inserter extends wp.element.Component {
 					icon="insert"
 					label={ wp.i18n.__( 'Insert block' ) }
 					onClick={ this.toggle }
-					className="editor-inserter__toggle" />
-				{ opened && <InserterMenu position={ position } onSelect={ this.close } /> }
+					className="editor-inserter__toggle"
+					aria-haspopup="true"
+					id="inserter-toggle"
+					buttonRef={ ( node ) => this.nodes.toggle = node }
```

Comment:
I'm suspecting the behaviour here won't be ideal after the rebase and the "clickOutside" behaviour. Do we really want to refocus the inserter button if we click outside?

But I'm trying to think what could we do here. Maybe we should transform the `IconButton` component into a class component instead of a functional component to allow `ref` on it?